e2e/shortcut-action/shortcut-action.e2e-spec.ts

```diff
@@ -86,18 +106,19 @@ describe('ShortcutAction', function() {
 
   it(`should copy new row under keywords.0 table using 'alt+c' shortcut. 
     The copied cell under the focused one must be empty and the remaining cells must be exactly copied.`, () => {
-    let firstColInputElem = page.getChildOfElementByCss(page.getElementById('keywords.0.classification_scheme'), 'textarea');
-    let secondColInputElemValue = page.getChildOfElementByCss(page.getElementById('keywords.0.keyword'), 'textarea')
-      .getAttribute('value');
-    firstColInputElem.sendKeys(protractor.Key.chord(protractor.Key.ALT, 'c'));
-    let firstColAfterCopyInputElemValue = page
-      .getChildOfElementByCss(page.getElementById('keywords.1.classification_scheme'), 'textarea')
-      .getAttribute('value');
-    let secondColAfterCopyInputElemValue = page.getChildOfElementByCss(page.getElementById('keywords.1.keyword'), 'textarea')
-      .getAttribute('value');
-    expect(firstColAfterCopyInputElemValue).toEqual('');
-    expect(secondColAfterCopyInputElemValue).toEqual(secondColInputElemValue);
+    let currentRowPromise = page.getValuesOfChildrenById('keywords.0');
```

Comment:
`currentRowPromise` is a good name since a promise is returned. On the other hand, only this test uses that naming convention, can you update the other ones to use the same naming?